### PR TITLE
replaced  ```rust with  ```cairo

### PR DESCRIPTION
**Pull request by nnlgsakib**

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ There are various `Source` implementation for different methods of downloading p
 2. `GitSource` downloads packages from Git repositories.
 3. `RegistrySource` downloads packages from package registries.
 4. `StandardLibSource` unpacks packages embedded into the Scarb binary itself.
-4. And more...
```
Comment (maciektr) [inline ARCHITECTURE.md:191]: I don't think this is correct 🤔

File: MAINTAINING.md [hunk 2]
```diff
@@ -64,9 +64,9 @@ We take an inspiration from the awesome release notes that [Visual Studio Code][
 Welcome to the release notes for Scarb vX.Y.Z!
 This release is all about blahblah
 
-* **Blah blah** - Why this blah is so much blah.
-* **Even more blah blah** - Everything is awesome.
-    * **Sometimes some sub-blah is cool** - Yeah.
+- **Blah blah** - Why this blah is so much blah.
+- **Even more blah blah** - Everything is awesome.
```
Comment (maciektr) [inline MAINTAINING.md:68]: Can you please avoid changing things like this?

File: website/docs/reference/procedural-macro.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ accept token streams.
 
 Example:
 
-```rust
+```cairo
```
Comment (maciektr) [inline website/docs/reference/procedural-macro.md:146]: That's actually Rust :)

File: design/01-proc-macro.md [hunk 4]
```diff
@@ -153,7 +157,7 @@ version = "0.1.0"
 [cairo-plugin]
 ```
 
-```rust
+```cairo
```
Comment (maciektr) [inline design/01-proc-macro.md:160]: This is actually Rust :)

**Review by maciektr — COMMENTED** [review]
Hi!

Can you please avoid reformatting all MD files while doing this? 
It makes the change set less obvious with loads of unconnected changes. 

CC @wawel37 for scarb-doc review

**Comment by nnlgsakib** [comment]
I am closing this

**Pull request closed without merging**